### PR TITLE
Assistant bot documentation

### PR DESCRIPTION
**Pull request by desrever-nu**
Added a section to document assistant bot

### CODE REVIEW

File: _kb/nbt-discourse-assistantbot.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+As well as helping in the formatting of motions , the bot can also fetch any of the NuNet information available in the Nu client and present it here in any public topic. Things like the current liquidity, the current difficulty, the current state of a particular motion or the current park rates can all be accessed with a simple command.
+Because the bot is hooked into the Nu Network, it can also let any user tip NuBits to any other user on this forum. To get started with tipping, send a PM with the word 'register' to [assistant](https://discuss.nubits.com/users/assistant/activity) . If you send someone who hasn't registered a tip, I'll create them an account and let them know a tip is waiting for them.
+
+To help get started, below  there is a list of the commands the bot will respond to. Please note that the bot will only respond to 'Private Message' commands via private message. Mention commands will only work when you mention its name in a forum post and type the command as shown below.
+
+## Private Message Commands
```
Comment (brossi) [inline _kb/nbt-discourse-assistantbot.md:19]: I noticed a possible inconsistency with how the section headings are declared. For example, this is an H2, but the next two sections, "Formatting Commands" and "Tipping Private Message Commands" are H3s. The section after that, "Mention Commands" are back to an H2.

Are they intentionally different or is it an oversight?

File: _kb/nbt-discourse-assistantbot.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- `@assistant motion votes` : Get the latest Motion Votes
+- `@assistant motion vote [motion_hash]` : Get the vote details for the motion with hash [motion_hash]
+- `@assistant park rates` : Get the latest Park Rate values
+- `@assistant park rate votes` : Get the latest Park Rate Votes
+- `@assistant verify` : Scan the thread for motions and custodial proposals and verify that their hashes are unchanged
+- `@assistant qrcode [address]` : Create a QR Code of the valid NBT address.
+- `@assistant tip [amount] @[user]` : Send a tip of [amount] NBT to @[user].
+
+
+## Standardise motion texts
```
Comment (brossi) [inline _kb/nbt-discourse-assistantbot.md]: This should probably read "Standardised Motion Texts" to keep the heading's title case in line with the rest of the site.

My American upbringing almost forced me to correct "standardised" to "standardized" but then I thought about it and came to the conclusion that I appreciate the global "flavor". However, we'll need to be careful that it doesn't distract from what we're trying to say if there are different spellings of the same word in the same document (not an issue in this case, just a comment).

File: _kb/nbt-discourse-assistantbot.md (hunk 1 above)
Comment (desrever-nu) [inline _kb/nbt-discourse-assistantbot.md:19]: Is intentional as they are hierarchically subtitles. Do you prefer to change them to a flat structure?

File: _kb/nbt-discourse-assistantbot.md (hunk 2 above)
Comment (desrever-nu) [inline _kb/nbt-discourse-assistantbot.md]: will update soon to 
Standardised Motion Texts

File: _kb/nbt-discourse-assistantbot.md (hunk 1 above)
Comment (brossi) [inline _kb/nbt-discourse-assistantbot.md:19]: No, that's fine, I just wanted to confirm that is what was intended.

**Pull request merged**